### PR TITLE
Update localized region names

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by jcristau** [comment]
@flodolo can you remind me the source we use for these names?  (We should document that somewhere if it's not already written up)

**Comment by flodolo** [comment]
The script is https://gist.github.com/flodolo/5bf943d88f9a5d2725789728aae077cd

It's taking translations from regionNames.ftl in separate l10n repos. This is the [en-US file](https://searchfox.org/mozilla-central/source/toolkit/locales/en-US/toolkit/intl/regionNames.ftl).

File: public/1.0/regions/index.html [hunk 1]
```diff
@@ -6,116 +7,117 @@
...
-      <li><a href="uk.json"> uk.json</a></li>
-      <li><a href="ur.json"> ur.json</a></li>
-      <li><a href="uz.json"> uz.json</a></li>
-      <li><a href="vi.json"> vi.json</a></li>
-      <li><a href="wo.json"> wo.json</a></li>
-      <li><a href="xh.json"> xh.json</a></li>
-      <li><a href="zh-CN.json"> zh-CN.json</a></li>
-      <li><a href="zh-TW.json"> zh-TW.json</a></li>
+      <li><a href="/1.0">Parent Directory</a></li>
+      <li><a href="en-US.json">en-US</a></li>
```
Comment (jcristau) [inline public/1.0/regions/index.html:11]: To double check, are the reordering (putting en-US first) and removal of `.json` from the text intentional?

**Review by jcristau — APPROVED** [review]

File: public/1.0/regions/index.html (hunk 1 above)
Comment (flodolo) [reply public/1.0/regions/index.html:11]: I personally don't see much value in spelling out `.json` in the anchor's text, but I can restore it if needed.

As for the reordering, I actually didn't notice I was moving en-US at the top. Fixing it.

Comment (jcristau) [reply public/1.0/regions/index.html]: I don't feel strongly about the `.json` so I'll merge this as-is.  Thanks!

**Comment by jcristau** [comment]
For future reference: it turns out `public/1.0/regions/index.html` is generated by shipit, so the changes from this PR ended up getting reverted.

**Pull request merged**